### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ryan-roemer**
- Update / harmonize various dependencies
- Switch to formidable lint
- Update travis

/cc @boygirl

### CODE REVIEW

File: config/eslint/.eslintrc-test [hunk 1]
```diff
@@ -11,6 +11,8 @@ plugins:
 
 rules:
   mocha/no-exclusive-tests: 2
+  no-magic-numbers: 0
+  import/no-unresolved: 0 # We have custom `src` webpack prefix that is not node-resolvable
```
Comment (boygirl) [inline config/eslint/.eslintrc-test:15]: Should we keep a separate `.eslintrc-demo` or just extend `.eslintrc-source` and add rules?

**Comment by boygirl** [comment]
approved pending potentially adding `.eslintrc-demo`

File: config/eslint/.eslintrc-test (hunk 1 above)
Comment (ryan-roemer) [reply config/eslint/.eslintrc-test:15]: We can do either, but I'd prefer follow-on work as this is mostly a "make it still work in spite of dependency updates" release 😉

Comment (boygirl) [reply config/eslint/.eslintrc-test:15]: works for me

**Comment by ryan-roemer** [comment]
Released in `builder-victory-component@4.0.0`

**Pull request merged**